src/ol/format/MVT.js

```diff
@@ -440,4 +445,28 @@ function getGeometryType(type, numEnds) {
   return geometryType;
 }
 
+/**
+ * Determines Flat coordinates is clockwise in MVT.
+ * @param {Array<number>} flatCoordinates Flat coordinates.
+ * @param {number} offset Offset.
+ * @param {number} end End.
+ * @param {number} stride Stride.
+ * @return {boolean} Is clockwise in MVT.
+ */
+function linearRingIsClockwise(flatCoordinates, offset, end, stride) {
```

Comment:
Couldn't you just `NOT` the result of the imported `linearRingIsClockwise` function, instead of defining a new one here?

Reply:
I have thought about "imported linearRingIsClockwise function",But I am confused about the case where it is equal to 0. when the result is 0,the `orient.linearRingIsClockwise ` will return `false`,but it is not counter-clockwise in mvt either.